hero html, hero css

26Georgiy opened the pull request.

26Georgiy committed:
hero html, hero css

vikadmin88 commented on src/css/hero.css:
```diff
@@ -0,0 +1,435 @@
+  align-items: flex-end;
+  justify-content: center;
+}
+.hero-img {
```
добав margin: 0 auto;  це відцентрує фотку без флекса

vikadmin88 commented on src/css/hero.css:
```diff
@@ -0,0 +1,435 @@
+  }
+}
+
+.hero-img-desktop {
```
забери все звідси
центруємо маржином у класі імаджа

vikadmin88 commented on src/css/hero.css:
```diff
@@ -0,0 +1,435 @@
+  font-size: 40px;
+  font-weight: 500;
+  line-height: 1.1;
+  letter-spacing: 0.04em;
```
letter-spacing: 0.04em;  зроби з мінусом. я тобі писав про те що я помилився коли порадив тобі забрати мінус
Текст стане ідеально і картинка по висоті вирівняється

26Georgiy committed:
css changes

26Georgiy committed:
json update

26Georgiy committed:
Merge branch 'main' into section-hero

26Georgiy committed:
html, css code-changes

HTML changes:
1) "happy students", arrow-icon is one a-element;
2) star-icon isn't a-element;
3) hero-image has width, height in html-code, deleted from css rule;
Css changes:
1) hero-title-h
2) hero-title-p
3) hero-title-span
4) hero-arrow
5) hero-star
6) hero-img
7) star-icon (mobile rules css) - deleted.

26Georgiy committed:
icon changes
